extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1175,15 +1216,23 @@ export class AmpA4A extends AMP.BaseElement {
       'CLICK_Y': () => {
         return e.pageY;
       },
+      'POSTMESSAGE': (origin, varName) => {
```

Comment:
Are not going to make this genetic to any postmessage?
- If we are, then there are other security issues to be considered. 
- If not, I suggest making it `BG_SIGNAL(type)`, which is more specific and reduces chance of mis-use.